xml/ay_networking.xml

```diff
@@ -19,51 +19,73 @@
-    will keep network settings created during the installation (via
-    <command>linuxrc</command>) and/or merge it with network settings from the
-    &ay; control file (if defined).
+    This network configuration is usually done through <command>linuxrc</command>
    </para>
+
    <note>
     <title>The <command>linuxrc</command> program</title>
     <para>
      For a detailed description of how <command>linuxrc</command> works and its
      keywords, see <xref linkend="appendix-linuxrc"/>.
     </para>
    </note>
+
+   <para>
+    By default &yast; keeps the network settings created during the installation.
+    Thus, linuxrc configured network could be considered the minimal network
+    configuration.
+   </para>
    <para>
-    &ay; settings have higher priority than any configuration files already
-    present. &yast; will write ifcfg-* files based on the entries in the control
-    file without removing old ones. If there is an empty or absent DNS and
-    routing section, &yast; will keep any pre-existing values. Otherwise,
-    settings from the control file will be applied.
+    The copy of the configuration can be ommited setting the 
+    <literal>keep_install_network</literal> option to <literal>false</literal>.
    </para>
 
 <screen>&lt;keep_install_network
-config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
+config:type="boolean"&gt;false&lt;/keep_install_network&gt;</screen>
 
    <para>
-    During the second stage, installation of additional packages will take
-    place before the network, as described in the profile, is configured.
-    <literal>keep_install_network</literal> is set by default to
-    <literal>true</literal> to ensure that a network is available
-    in case it is needed to install those packages. If all packages
-    are installed during the first stage and the network is not needed early
-    during the second one, setting <literal>keep_install_network</literal>
-    to <literal>false</literal> will avoid copying the configuration.
+    In case of kept, the settings will be merged with the network settings
+    from the &ay; control file. &ay; settings have higher priority than any
+    configuration files already present. &yast; will write ifcfg-* files based
+    on the entries in the control file without removing old ones. If there is
+    an empty or absent DNS and routing section, &yast; will keep any pre-existing
+    values. Otherwise, settings from the control file will be applied.
    </para>
 
+   <note>
+     <title>Use &ay; network settings during installation</title>
+
+    <para>
+      Since &productname; 15.3, the write of the configuration based on the
+      control file will happen at the end of the first stage by default.
+    </para>
+    <para>
+      However, if the &ay; network settings are needed during the installation
+      it can be forced to be written and applied before the registration takes
```

Comment:
Remove the first *&ay* and rephrase the second part as *you can ask/force &ay; to write and apply them before...".